core/codenvy-hosted-platform-api-impl/src/main/java/com/codenvy/api/workspace/LimitsCheckingWorkspaceManager.java

```diff
@@ -169,21 +171,26 @@ public WorkspaceImpl updateWorkspace(String id, Workspace update) throws Conflic
         final Lock lock = START_LOCKS.get(user);
         lock.lock();
         try {
-            final long currentlyUsedRamMB = getWorkspaces(user).stream()
-                                                               .filter(ws -> STOPPED != ws.getStatus())
-                                                               .map(ws -> ws.getConfig()
-                                                                            .getEnvironment(ws.getRuntime().getActiveEnv())
-                                                                            .get()
-                                                                            .getMachineConfigs())
-                                                               .mapToLong(this::sumRam)
-                                                               .sum();
+            final List<WorkspaceImpl> workspacesPerUser = getWorkspaces(user);
+            final long runningWorkspaces = workspacesPerUser.stream().filter(ws -> RUNNING == ws.getStatus()).count();
+            final long currentlyUsedRamMB = workspacesPerUser.stream().filter(ws -> STOPPED != ws.getStatus())
+                                                             .map(ws -> ws.getConfig()
+                                                                          .getEnvironment(
```

Comment:
change the formatting back please `.getEnvironment(ws.getRuntime().getActiveEnv())`